rinsar/_process_utilities.py

```diff
@@ -418,24 +418,25 @@ def file_len(fname):
 ##########################################################################
 
 
-def alphanum_key(s):
-    """ Turn a string into a list of string and number chunks.
-        "z23a" -> ["z", 23, "a"]
-    """
-    return [ int(c) if c.isdigit() else c for c in re.split('([0-9]+)', s) ]
+#def alphanum_key(s):
```

Comment:
Could you delete these commented-out functions? A good principle is that no commented-out code should be committed